### PR TITLE
Create hh tasks for reworked aggregator

### PR DESCRIPTION
**Pull request by halaprix**
- create -`create-trigger=group-cm` task
- create `remove-triggers` task

### CODE REVIEW

File: scripts/tasks/remove-triggers.task.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            signer = await hardhatUtils.impersonate(currentProxyOwner)
+            // Fund the owner
+            await hre.ethers.provider.getSigner(0).sendTransaction({
+                to: currentProxyOwner,
+                value: EthersBN.from(10).pow(18),
+            })
+        }
+
+        const removeTriggerGroupData = aggregator.interface.encodeFunctionData('removeTriggers', [
+            [...args.triggers.map(item => item.toString())],
```
Comment (rkrasiuk) [inline scripts/tasks/remove-triggers.task.ts:54]: unnecessary spread

**Review by rkrasiuk — CHANGES_REQUESTED** [review]

File: scripts/tasks/create-trigger-group-cm.task.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            console.log(`Impersonating proxy owner ${currentProxyOwner}...`)
+            signer = await hardhatUtils.impersonate(currentProxyOwner)
+            // Fund the owner
+            await hre.ethers.provider.getSigner(0).sendTransaction({
+                to: currentProxyOwner,
+                value: EthersBN.from(10).pow(18),
+            })
+        }
+
+        const bbTriggerData = encodeTriggerData(args.vault.toNumber(), TriggerType.CM_BASIC_BUY, ...args.bb)
```
Comment (adamskrodzki) [inline scripts/tasks/create-trigger-group-cm.task.ts:75]: TriggerType.CM_BASIC_BUY can be changed back to  TriggerType.BASIC_BUY

File: scripts/tasks/create-trigger-group-cm.task.ts [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        const bsTriggerData = encodeTriggerData(args.vault.toNumber(), TriggerType.CM_BASIC_SELL, ...args.bs)
+
+        const triggersData = [bbTriggerData, bsTriggerData]
+        const counter = await bot.triggersCounter()
+        const addTriggerGroupData = aggregator.interface.encodeFunctionData('addTriggerGroup', [
+            args.type.toString(),
+            [0, 0],
```
Comment (adamskrodzki) [inline scripts/tasks/create-trigger-group-cm.task.ts:82]: can we add parameter here and [0,0] make default ?

**Review by adamskrodzki — CHANGES_REQUESTED** [review]

File: scripts/tasks/create-trigger-group-cm.task.ts (hunk 3 above)
Comment (halaprix) [reply scripts/tasks/create-trigger-group-cm.task.ts:82]: done

File: scripts/tasks/create-trigger-group-cm.task.ts (hunk 2 above)
Comment (halaprix) [reply scripts/tasks/create-trigger-group-cm.task.ts:75]: ok. I'm changing it and I'll send the `revert all the rest of the repo to 3 and 4` PR in a sec

**Review by rkrasiuk — APPROVED** [review]

**Pull request merged**